shunting-yard.h

```diff
@@ -159,6 +159,9 @@ class Function;
 // as well as some built-in functions:
 #include "./functions.h"
 
+// Global counter for detecting unicode characters
+static unsigned char unicodeCharCounter = 0;
```

Comment:
We can't use global variables for this. This would make it impossible to run 2 parsers in parallel, and we should be able to implement this keeping the state some other way.